<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>widget_reference_book</class>
 <widget class="QWidget" name="widget_reference_book">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="IconButton" name="back_to_menu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/left.png</normaloff>:/button/resources/button/left.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="resources.qrc">:/background/resources/background/first_background.png</pixmap>
   </property>
  </widget>
  <widget class="IconButton" name="item1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item2">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item3">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>100</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item5">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>210</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item8">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>320</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item9">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>320</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item10">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>430</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="IconButton" name="item12">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>430</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
        background-color: rgba(255, 255, 255,128);
        border-radius: 15px; /* 圆角（可选） */
    }</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/resource/resources/resource/Inspiration.png</normaloff>:/resource/resources/resource/Inspiration.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <attribute name="buttonGroup">
    <string notr="true">buttonGroup</string>
   </attribute>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>100</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::Box</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="description">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>410</y>
     <width>251</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'幼圆'; font-size:12pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="value1">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>370</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:white;  
       

border-radius: 6px;       
padding: 5px;            </string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::SegmentStyle::Flat</enum>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLCDNumber" name="value2">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>370</y>
     <width>51</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"> background-color:white;  
       

border-radius: 6px;       
padding: 5px;            </string>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="smallDecimalPoint">
    <bool>true</bool>
   </property>
   <property name="segmentStyle">
    <enum>QLCDNumber::SegmentStyle::Flat</enum>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>价 格</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>360</y>
     <width>41</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;</string>
   </property>
   <property name="text">
    <string>伤 害</string>
   </property>
  </widget>
  <widget class="IconButton" name="upgrade">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>350</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/upgrade.png</normaloff>:/button/resources/button/upgrade.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <widget class="IconButton" name="next">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>480</y>
     <width>65</width>
     <height>65</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/button/resources/button/right.png</normaloff>:/button/resources/button/right.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>70</width>
     <height>70</height>
    </size>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>back_to_menu</zorder>
  <zorder>item1</zorder>
  <zorder>item2</zorder>
  <zorder>item3</zorder>
  <zorder>item4</zorder>
  <zorder>item5</zorder>
  <zorder>item6</zorder>
  <zorder>item7</zorder>
  <zorder>item8</zorder>
  <zorder>item9</zorder>
  <zorder>item10</zorder>
  <zorder>item11</zorder>
  <zorder>item12</zorder>
  <zorder>label</zorder>
  <zorder>description</zorder>
  <zorder>value1</zorder>
  <zorder>value2</zorder>
  <zorder>label1</zorder>
  <zorder>label2</zorder>
  <zorder>upgrade</zorder>
  <zorder>next</zorder>
 </widget>
 <customwidgets>
  <customwidget>
   <class>IconButton</class>
   <extends>QPushButton</extends>
   <header>IconButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
